SwanHeader/style/base.css

```diff
@@ -0,0 +1,86 @@
+#swan-header {
+  --swan-color-dark-1: #09263c;
```

Comment:
Fine for now. Eventually we should create a separate extension (used as dependency) with these tokens and the logos.